docs/source/configuration/overview.mdx

```diff
@@ -166,6 +166,25 @@ The default value is `info`.
 <tr>
 <td style="min-width: 150px;">
 
+##### `--license`
+
+`APOLLO_ROUTER_LICENSE_PATH`, `APOLLO_ROUTER_LICENSE`
+
+</td>
+<td>
+
+An offline GraphOS Enterprise license. Enables Enterprise router features when disconnected from GraphOS. 
```

Comment:
Since it accepts a path via the `APOLLO_ROUTER_LICENSE_PATH` option, maybe include "The absolute or relative path to the Apollo Router's license"?